src/test/kotlin/pt/up/fe/ni/website/backend/controller/EventControllerTest.kt

```diff
@@ -309,32 +313,44 @@ internal class EventControllerTest @Autowired constructor(
     @NestedTest
     @DisplayName("POST /events/new")
     inner class CreateEvent {
+        private val uuid: UUID = UUID.randomUUID()
+        private val mockedSettings = Mockito.mockStatic(UUID::class.java)
+        private val expectedImagePath = "${uploadConfigProperties.staticServe}/events/${testEvent.title}-$uuid.jpeg"
+
         @BeforeEach
         fun addAccount() {
             accountRepository.save(testAccount)
         }
 
+        @BeforeAll
+        fun setupMocks() {
+            Mockito.`when`(UUID.randomUUID()).thenReturn(uuid)
+        }
+
+        @AfterAll
+        fun cleanup() {
+            mockedSettings.close()
+        }
+
```

Comment:
I think this setup is cleaner, maybe you can apply this to the account tests too?

Reply:
Sure thing!